task9/exception/ValidationException.js

```diff
@@ -0,0 +1,9 @@
+import Exception from "./Exception.js";
+
+class ValidationException extends Exception {
```

Comment:
do we really need new ValidationException? Exception should be enough

Reply:
But you have 1 exception filter (error handler) which will expect limited amount of error types

Reply:
I've check exception filter and it is working for ValidationException pretty well(I debugged it), cause I also was curious will it we working or not
![image](https://github.com/user-attachments/assets/6f68af2a-5acf-43d6-86ef-f1a0d5866176)

btw, I can simplify ValidationException to this implementation:

```
import Exception from './Exception.js';

class ValidationException extends Exception {
}

export default ValidationException;
```

maybe I miss smth, but errorHandler is catching Error(Exception extends Error, ValidationException extends Exception, according to transitivity property ValidationException extends Error)

Reply:
fixed